build/make/Makefile.in

```diff
@@ -130,15 +130,6 @@ PIP_PACKAGES = @SAGE_PIP_PACKAGES@
 # Packages that use the 'script' package build rules
 SCRIPT_PACKAGES = @SAGE_SCRIPT_PACKAGES@
 
-# Packages for which we build wheels for PyPI
-PYPI_WHEEL_PACKAGES = \
```

Comment:
This is still referenced in a few places (eg 2 lines below). Please remove these as well.

Reply:
yes, I was wondering whether to remove sagemath_categories, etc., there 

Should I?